M6-Backend/MockFacility.test.js

```diff
@@ -0,0 +1,210 @@
+//Test Set 1 --> Testing the interface specific for module facility
+const interfacespecific = require('./interfacespecific');
+
+test("interfacespecific invalid input", ()=>{
+  expect(interfacespecific("ll@@@@}}")).not.toEqual('posts');
+})
+
+test("interfacespecific number out of bound", ()=>{
+  expect(interfacespecific(100000)).not.toEqual('studys');
```

Comment:
Codacy has a fix for the issue: [Replace `'studys'` with `"studys"`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
  expect(interfacespecific(100000)).not.toEqual("studys");
```